src/main/java/duke/task/TaskList.java

```diff
@@ -67,7 +67,7 @@ public boolean isIndexValid(String index) {
      * element in the taskList. This function returns all the names of tasks (from their respective
      * toString() functions) as an array.
      *
-     * @return An array containing the names of every task in the taskList.
+     * @return An array containing the names of every duke.task in the taskList.
```

Comment:
Hi Sam, disregard this if this was your intention. But, perhaps reflecting the package of task in this comment might be unnecessary and possible a result of refactoring when you packaged your classes. Consider inspecting and removing the "duke." in "duke.task", perhaps?

Reply:
This was indeed the result of a rogue refactor, nice catch!